be/src/Project/Services/Catalog/Catalog.Api/Services/Products/ProductsService.cs

```diff
@@ -147,25 +158,25 @@ public async Task<bool> IsEmptyAsync()
         {
             using var source = new ActivitySource(this.GetType().Name);
 
-            var brand = catalogContext.Brands.FirstOrDefault(x => x.SellerId == model.OrganisationId.Value && x.IsActive);
+            var brand = _context.Brands.FirstOrDefault(x => x.SellerId == model.OrganisationId.Value && x.IsActive);
 
-            if (brand == null)
+            if (brand is not null)
```

Comment:
**suggestion:** should be: 

`if (brand is null)`

Reply:
I rushed too much with this change